handshake_docs/handshake_flow.md

```diff
@@ -0,0 +1,35 @@
+# Handshake Flow for DIDComm Mediator-rs
+
+## 1. Communication Initiation
+### Participant Identification:
+- Each mobile agent generates a DID (Decentralized Identifier).
+
```

Comment:
A mobile agent is called an edge agent.

Generally, the generation of a did involves some interaction with a mediator. This is the cloud agent that is responsible for:

1. Receiving messages sent to the edge agent
2. Forwarding messages sent out by the edge agent.

Reply:
Okay sir